client/src/components/Input.js

```diff
@@ -16,31 +17,37 @@ const SmallInput = styled.input`
   text-align: center;
 `;
 
-const LargeInput = styled.input`
-  height: 40rem;
+const LargeInput = styled.textarea`
+  height: 20rem;
   border-radius: 15px;
   width: 80%;
   border: solid 1px var(--secondary-font-color);
-  text-align: center;
 `;
```

Comment:
Are these specific to the Form or can they be used in other components as well? Maybe create a separate component for these.

Reply:
they are specific to this form :)